src/invert/laplace/impls/mumps/mumps_laplace.cxx

```diff
@@ -549,91 +550,94 @@ LaplaceMumps::LaplaceMumps(Options *opt) :
 // 
 // }
 
-const FieldPerp LaplaceMumps::solve(const FieldPerp &b, const FieldPerp &x0) {
-  return solve(b);
+const FieldPerp LaplaceMumps::solve(const FieldPerp &b_in, const FieldPerp &x0) {
+  return solve(b_in);
 }
 
-const FieldPerp LaplaceMumps::solve(const FieldPerp &b) {
+const FieldPerp LaplaceMumps::solve(const FieldPerp &b_in) {
+  FieldPerp b = b_in;
```

Comment:
If you want `b` to be unique from `b_in` then you may need a `b.allocate()` to ensure they have different data blocks.